libminifi/src/core/RepositoryFactory.cpp

```diff
@@ -48,9 +50,15 @@ std::shared_ptr<core::Repository> createRepository(
   try {
     std::shared_ptr<core::Repository> return_obj = nullptr;
     if (class_name_lc == "flowfilerepository") {
+      std::cout << "creating flow" << std::endl;
```

Comment:
sorry I added that to be captured by a test, but I guess I forgot the outputstreamappender can redirect to stdout. I'll fix. Thanks for reminding me.